pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -435,6 +439,11 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 			}
 
 			glog.Infof(volumeToMount.GenerateMsgDetailed("MountVolume.WaitForAttach succeeded", ""))
+			resizeError := og.resizeFileSystem(volumeToMount, devicePath, mounter)
```

Comment:
nit: From a reader's perspectives, it seems like volume manager is trying to resize every time a volume is mounted, which is confusing. It'd be helpful to explain what it actually does.

Reply:
added a comment